app/src/main/res/values-v21/styles.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2020 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
```

Comment:
we are minSDK 21, this could go in `values/styles.xml`

Reply:
So that's what I thought but for some reason it screws up the buttons on 21 🤷‍♂  

![Screenshot 2020-03-13 at 17 00 08](https://user-images.githubusercontent.com/1773137/76643004-291e6880-654c-11ea-9298-15bbfe361711.png)

Reply:
It turns out I can use `backgroundTint` rather than `android:backgroundTint` in the main styles file and everything works as expected 🤦‍♂, whenever I use `android:backgroundTint` and 21 is when it comes back as red. I thought the prefix android was supposed to be used 21 onwards but I guess not.